raiden/storage/sqlite.py

```diff
@@ -792,9 +792,11 @@ def write_state_snapshot(
         # `to_checksum_address` is slow and is not necessary for our internal serialization.
         # FIXME: We should be able to adapt the serialization without this evil
         #        monkey patching, but right now there is no simple way to do it.
-        fields.to_checksum_address = to_hex
+        fields.to_checksum_address = bytes.hex  # type: ignore
+        fields.to_canonical_address = bytes.fromhex  # type: ignore
```

Comment:
Do we actually use `to_canonical_address` when serializing? I would have expected it only in deserialization.